test/future_tests.cpp

```diff
@@ -428,9 +428,9 @@ BOOST_AUTO_TEST_CASE(future_blocking_get_void_with_timeout) {
 
```

Comment:
There are some tests where the compilation currently fails, because futures are marked as [[nodiscard]]

Reply:
Thanks. I will fix those.